internal/tui/model_list.go

```diff
@@ -151,7 +222,36 @@ func (m ListModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 
 	case tea.KeyMsg:
 		kmap := m.KeyMap()
```

Comment:
You can bring this line down below the `if m.editInput.Focused()` block, since it is not used until after that.